handlers/main.yml

```diff
@@ -16,27 +16,32 @@
   become_user: "{{ pretalx_system_user }}"
 
 - name: Run pretalx migrations
-  command: python{{ pretalx_system_python_version }} -m pretalx migrate
+  command: "{{ pretalx_python }} -m pretalx migrate"
   become: true
   become_user: "{{ pretalx_system_user }}"
 
 - name: Compile pretalx messages
-  command: python{{ pretalx_system_python_version }} -m pretalx compilemessages
+  command: "{{ pretalx_python }} -m pretalx compilemessages"
   become: true
   become_user: "{{ pretalx_system_user }}"
 
 - name: Compile pretalx styles
-  command: python{{ pretalx_system_python_version }} -m pretalx regenerate_css
+  command: "{{ pretalx_python }} -m pretalx regenerate_css"
+  become: true
+  become_user: "{{ pretalx_system_user }}"
+
+- name: Compile pretalx javascript
```

Comment:
Hmm, I don't think this is quite correct. Instead, we can just replace the `collectstatic` and `compress` commands with `rebuild` – rebuild runs collectstatic, compress and also compilemessages.